python/src/LogNormalMuErrorFactor_doc.i.in

```diff
@@ -0,0 +1,35 @@
+%feature("docstring") OT::LogNormalMuErrorFactor
+"LogNormal distribution parameters.
+
+Parameters
+----------
+mu : float
+    The mean of the log-normal random variable.
+ef : float
+    The error factor.
+gamma : float, optional
+    Location parameter.
+
+Notes
+-----
+The :math:`EF` parameter is defined as:
+
+.. math::
+
+    EF = \sqrt{\frac{q_{95} - \gamma}{q_{5} - \gamma}} = \exp{\sigma_{l} q_{95}^N}
```

Comment:
I suppose `q_{95}^N}`referes to centered & reduced gaussian quantile, wheras `q_{95}`refers to `log-normal`one.
Am I wrong ? Is it possible to add more details ? thanks

Reply:
done